tools/clang/tools/dxcompiler/dxcompilerobj.cpp

```diff
@@ -53,6 +53,8 @@
 #include "dxillib.h"
 #include <algorithm>
 
+#include "GitCommitInfo.inc" // Auto generated file containing Git commit info
```

Comment:
We should have the ability to leverage the preprocesseor revert to the prior implementation (no git checks, no commit information available) for cases where this isn't meaningful for downstream implementations, or actively confusing (because, for example, this ends up over in a different git repo altogether). I.e., have a way to ifdef this out.

Reply:
SG. I'll add a new CMake option to wrap both the CMake configuration and this around.

Reply:
Done